### PR TITLE
testcase/le_tc/network : add scenario for net_dup

### PR DESCRIPTION
**Pull request by arvinmittal**
file path : apps/examples/testcase/le_tc/network
file name : tc_net_dup.c

add scenario :
tc_net_dup_multiple_time_p
tc_net_dup2_multiple_time_p

Signed-off-by: Arvin Mittal <arvin.mittal@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check Result:
apps/examples/testcase/le_tc/network/tc_net_dup.c:139: ERROR: [SPC_M_OPR] space required after that ';' (ctx:VxV)
apps/examples/testcase/le_tc/network/tc_net_dup.c:176: ERROR: [SPC_M_OPR] space required after that ';' (ctx:VxV)

**Comment by seinfra** [comment]
Code Rule Check OK.

File: apps/examples/testcase/le_tc/network/tc_net_dup.c [hunk 1]
```diff
@@ -1,130 +1,202 @@
...
-#include <tinyara/config.h>
-#include <stdio.h>
-#include <errno.h>
-#include <sys/stat.h>
-#include <net/if.h>
-#include <arpa/inet.h>
-#include <netinet/in.h>
-#include <netutils/netlib.h>
-#include <sys/socket.h>
+*
```
Comment (sunghan-chang) [inline apps/examples/testcase/le_tc/network/tc_net_dup.c]: Your team modified this in almost pull request. Don't do that. Let's revert these kind of change.

File: apps/examples/testcase/le_tc/network/tc_net_dup.c [hunk 2]
```diff
@@ -1,130 +1,202 @@
...
+* @brief            :clone a file descriptor to arbitrary descriptor
+* @scenario         :dup file descriptor with invalid socket descriptor range
+* @apicovered       :dup()
+* @precondition     :none
+* @postcondition    :none
+*/
 static void tc_net_dup_n(void)
 {
-	/* the sock value is over valid socket descriptors range */
+	/* the sock value is over valid socket descriptors range*/
```
Comment (sunghan-chang) [inline apps/examples/testcase/le_tc/network/tc_net_dup.c]: ?

**Comment by sunghan-chang** [comment]
@arvinmittal @samegautam @ravisankarg @dubeyalok7 @kavyasrinunna @ktejram @yc207-lee 
Please be careful of below on making a pull reqeust.
1. Add "static" for local functions and variables which defines outside of function. This is a Tizen Coding Rule. You should follow it.
2. Don't use "git pull" to update yours to latest. It makes unnecessary / unacceptable commits. Use git rebase.
i.e.
```
Merge master to xxx
```
3. Don't modify code comments. Below is our rule.
```
/*****
 *
 ****/
```
4. Don't mix different kinds of changes. Formmating (or cosmetic) and changing codes should be splitted.
  Changing existed utc codes and adding new itc should be splitted.
5. In commit message, just writing function name is not acceptable. That is not helpful to user. Please write it to understand what you do. I don't want to guess from function name.

**Pull request closed without merging**